docs/QuickStart-Videos.md

```diff
@@ -7,14 +7,16 @@ permalink: docs/videos.html
 next: guides-containers
 ---
 
-## React.js Conf
+## [React.js Conf January 2015](http://conf.reactjs.com/)
```

Comment:
This is great! Do you mind also switching the order to put most-recent videos first?